lib/phoenix/live_dashboard/templates/layout/live.html.leex

```diff
@@ -1,14 +1 @@
-<header class="d-flex">
-  <div class="container d-flex flex-column">
-    <h1>
-      <span class="header-title-part">Phoenix </span>
-      <span class="header-title-part">LiveDashboard<span>
-    </h1>
-    <%= live_render @socket, Phoenix.LiveDashboard.MenuLive,
-         id: "menu", session: %{"menu" => %{@menu | info: nil}} %>
-  </div>
-</header>
-<%= live_info(@socket, @menu) %>
-<section id="main" role="main" class="container">
-  <%= @inner_content %>
-</section>
+<%= @inner_content %>
```

Comment:
I moved all this template to `PageLive`, I didn't figure out a better way to solve it